includes/class-api.php

```diff
@@ -388,6 +406,20 @@ public function sync_attachment( $attachment_args, $duplicate_action = 'skip', $
 			return false;
 		}
 
+		if ( ! empty( $this->ps_meta_repair_fields ) ) {
```

Comment:
Same comment as previous.

Reply:
The way both of these methods is structured doesn't allow us to reverse this just yet. It could at some point, but right now these "short-circuit" the login in the enclosing method and return early, otherwise the functions continue and do a lot of other work.

Reply:
This comment still shows for the similar line, however this is moved out from where it was earlier.